api/grant.go

```diff
@@ -47,6 +47,7 @@ func (r ListGrantsRequest) ValidationRules() []validate.ValidationRule {
 		validate.MutuallyExclusive(
 			validate.Field{Name: "user", Value: r.User},
 			validate.Field{Name: "group", Value: r.Group},
+			// TODO: validate showInherited only allowed with User
```

Comment:
I'll address this TODO in a follow up PR, since this one is already a good size.

Reply:
Addressed this in #3176